CONFIG.example

```diff
@@ -1,23 +1,23 @@
 # Example configuration file for building PandA firmware, and for simulation
 #
 # Copy this file to a file named CONFIG and edit as appropriate.  Lines that are
-# commented out can be left as they are, the default value is show.  Uncommented
+# commented out can be left as they are, the default value is shown.  Uncommented
 # lines must be assigned values.
 
 # Default build location.  Default is to build in build subdirectory.
 BUILD_DIR = /build
 
 # Development Tool Version
-VIVADO_VER = 2022.2
+# VIVADO_VER = 2023.2
```

Comment:
Why is this now commented?

Reply:
Just to make it consistent with the comments at the top of the file. According to this it should only be the uncommented variables that need to be set in the CONFIG, the others have default values in the Makefile.